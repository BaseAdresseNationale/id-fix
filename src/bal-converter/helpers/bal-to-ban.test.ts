import type { Bal } from '../../types/bal-types.js';

import fs from 'node:fs';
import { describe, expect, test } from 'vitest';
import balToBan from './bal-to-ban';

const pathToMockBalJSON = './data-mock/adresses-21286_cocorico.json';
const mockBalJSONstr = fs.readFileSync(pathToMockBalJSON, 'utf8');
const mockBalJSON: Bal = JSON.parse(mockBalJSONstr);

describe('balToBan', () => {
  test('should convert Bal list into Ban list', async () => {
    const banJSON = balToBan(mockBalJSON);
    expect(banJSON).toMatchSnapshot();
  });
});
