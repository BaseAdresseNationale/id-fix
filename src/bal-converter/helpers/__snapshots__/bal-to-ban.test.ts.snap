// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`balToBan > should convert Bal list into Ban list 1`] = `
{
  "addresses": {
    "03fb190a-cf5b-4f48-a1ab-7caa4d10e157": {
      "certified": false,
      "commonToponymID": "787ca7cf-8072-47ae-a8c6-98a62a8dd90c",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "03fb190a-cf5b-4f48-a1ab-7caa4d10e157",
      "number": 1,
      "parcels": [
        "212860000C0115,212860000C0114",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.081243,
              48.769923,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
        {
          "geometry": {
            "coordinates": [
              1.081148,
              48.770269,
            ],
            "type": "Point",
          },
          "type": "logement",
        },
      ],
      "suffix": "",
      "updateDate": "2022-08-23T00:00:00.000Z",
    },
    "18510122-0ad8-49fe-bbae-ca484d8f4e01": {
      "certified": false,
      "commonToponymID": "7ce61747-d840-4019-97be-82dc0568619f",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "18510122-0ad8-49fe-bbae-ca484d8f4e01",
      "number": 31,
      "parcels": [
        "",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.08586,
              48.773833,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "1890569a-d05f-4cf9-b27b-9172c497023e": {
      "certified": false,
      "commonToponymID": "c9b6df77-638b-4b30-991e-71486a91ea95",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "1890569a-d05f-4cf9-b27b-9172c497023e",
      "number": 6,
      "parcels": [
        "212860000D0237",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.086775,
              48.759802,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "2480fe76-bbd8-4703-a41a-5c43fd3c5891": {
      "certified": false,
      "commonToponymID": "468d0b9b-ec0c-448d-8188-4e28762251e2",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "2480fe76-bbd8-4703-a41a-5c43fd3c5891",
      "number": 10,
      "parcels": [
        "212860000D0053,212860000D0054",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.082305,
              48.762015,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "28fd0d23-9f3a-4ac0-8e17-05f851546f34": {
      "certified": false,
      "commonToponymID": "468d0b9b-ec0c-448d-8188-4e28762251e2",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "28fd0d23-9f3a-4ac0-8e17-05f851546f34",
      "number": 5,
      "parcels": [
        "212860000D0305,212860000D0307,212860000D0308,212860000D0266",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.085079,
              48.762971,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "29387aa3-1dc6-4e38-a1e9-aea350c91296": {
      "certified": false,
      "commonToponymID": "7ce61747-d840-4019-97be-82dc0568619f",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "29387aa3-1dc6-4e38-a1e9-aea350c91296",
      "number": 33,
      "parcels": [
        "",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.085574,
              48.774035,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "48691fd6-a611-4335-b346-a05679bcb31d": {
      "certified": false,
      "commonToponymID": "c9b6df77-638b-4b30-991e-71486a91ea95",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "48691fd6-a611-4335-b346-a05679bcb31d",
      "number": 11,
      "parcels": [
        "212860000D0143",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.087958,
              48.760488,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "4afc82e4-694e-4bb3-81cb-0a32cd59a300": {
      "certified": false,
      "commonToponymID": "7ce61747-d840-4019-97be-82dc0568619f",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "4afc82e4-694e-4bb3-81cb-0a32cd59a300",
      "number": 29,
      "parcels": [
        "",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.086279,
              48.773553,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "5114f638-f99d-4da2-b3ed-d2e2c3a32765": {
      "certified": false,
      "commonToponymID": "468d0b9b-ec0c-448d-8188-4e28762251e2",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "5114f638-f99d-4da2-b3ed-d2e2c3a32765",
      "number": 9,
      "parcels": [
        "212860000D0290,212860000D0292,212860000D0273,212860000D0288",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.083096,
              48.76238,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "5139b74a-f569-4008-aade-3bb955186134": {
      "certified": false,
      "commonToponymID": "468d0b9b-ec0c-448d-8188-4e28762251e2",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "5139b74a-f569-4008-aade-3bb955186134",
      "number": 8,
      "parcels": [
        "21286000ZC0087",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.082967,
              48.762421,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "577e9019-42bb-4965-b287-fe4cebe41038": {
      "certified": false,
      "commonToponymID": "468d0b9b-ec0c-448d-8188-4e28762251e2",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "577e9019-42bb-4965-b287-fe4cebe41038",
      "number": 1,
      "parcels": [
        "212860000D0113",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.085624,
              48.763273,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "61b54625-bef6-4450-95e1-b6f37d93441a": {
      "certified": false,
      "commonToponymID": "468d0b9b-ec0c-448d-8188-4e28762251e2",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "61b54625-bef6-4450-95e1-b6f37d93441a",
      "number": 3,
      "parcels": [
        "212860000D0113",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.08535,
              48.763191,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "6edd2c77-e8db-4384-bbb6-810e868d4dee": {
      "certified": false,
      "commonToponymID": "c9b6df77-638b-4b30-991e-71486a91ea95",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "6edd2c77-e8db-4384-bbb6-810e868d4dee",
      "number": 3,
      "parcels": [
        "212860000D0304,212860000D0229",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.086349,
              48.759711,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "7a9e11ff-31b6-4f14-8494-55369e7dce04": {
      "certified": false,
      "commonToponymID": "468d0b9b-ec0c-448d-8188-4e28762251e2",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "7a9e11ff-31b6-4f14-8494-55369e7dce04",
      "number": 7,
      "parcels": [
        "212860000D0293",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.084526,
              48.76266,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "8c12024d-6216-4634-aca0-598f569d632b": {
      "certified": false,
      "commonToponymID": "c965715f-3874-4cba-ae4e-c9afa585f5eb",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "8c12024d-6216-4634-aca0-598f569d632b",
      "number": 2,
      "parcels": [
        "",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.079699,
              48.773594,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "aeda3e20-8204-4d2b-8e59-184df887ae4e": {
      "certified": false,
      "commonToponymID": "c965715f-3874-4cba-ae4e-c9afa585f5eb",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "aeda3e20-8204-4d2b-8e59-184df887ae4e",
      "number": 1,
      "parcels": [
        "",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.07932,
              48.773412,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
        {
          "geometry": {
            "coordinates": [
              1.078832,
              48.773161,
            ],
            "type": "Point",
          },
          "type": "logement",
        },
      ],
      "suffix": "",
      "updateDate": "2022-08-23T00:00:00.000Z",
    },
    "d19b14c9-d9d0-478f-b741-c973d05c343f": {
      "certified": false,
      "commonToponymID": "787ca7cf-8072-47ae-a8c6-98a62a8dd90c",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "d19b14c9-d9d0-478f-b741-c973d05c343f",
      "number": 2,
      "parcels": [
        "212860000C0237,212860000C0107",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.081444,
              48.769615,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "e6278261-1a1b-4606-9c20-a8f2d06738fa": {
      "certified": false,
      "commonToponymID": "c9b6df77-638b-4b30-991e-71486a91ea95",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "e6278261-1a1b-4606-9c20-a8f2d06738fa",
      "number": 8,
      "parcels": [
        "212860000D0236",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.087306,
              48.760129,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "e9de2ecf-e082-448b-9a21-fe9d38d017ae": {
      "certified": false,
      "commonToponymID": "c9b6df77-638b-4b30-991e-71486a91ea95",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "e9de2ecf-e082-448b-9a21-fe9d38d017ae",
      "number": 7,
      "parcels": [
        "212860000D0150",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.086974,
              48.760002,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "f1a28bf1-fac3-41f0-8cb4-3abc82b2b963": {
      "certified": false,
      "commonToponymID": "c9b6df77-638b-4b30-991e-71486a91ea95",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "f1a28bf1-fac3-41f0-8cb4-3abc82b2b963",
      "number": 5,
      "parcels": [
        "212860000D0230",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.086628,
              48.759821,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "ff8bb658-3f73-4ac3-ab04-7966e869e991": {
      "certified": false,
      "commonToponymID": "c9b6df77-638b-4b30-991e-71486a91ea95",
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "id": "ff8bb658-3f73-4ac3-ab04-7966e869e991",
      "number": 9,
      "parcels": [
        "212860000D0148,212860000D0284",
      ],
      "positions": [
        {
          "geometry": {
            "coordinates": [
              1.087483,
              48.760315,
            ],
            "type": "Point",
          },
          "type": "entrée",
        },
      ],
      "suffix": "",
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
  },
  "commonToponyms": {
    "16e337a0-d0c7-4c24-b8dc-e43edbd4d273": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.067195,
          48.760165,
        ],
        "type": "Point",
      },
      "id": "16e337a0-d0c7-4c24-b8dc-e43edbd4d273",
      "label": [
        {
          "isoCode": "fra",
          "value": "Les Bois Perdus",
        },
      ],
      "parcels": [
        "",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2022-06-03T00:00:00.000Z",
    },
    "1748660f-d473-400a-88ad-52d55f3fc9f5": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.063471,
          48.771215,
        ],
        "type": "Point",
      },
      "id": "1748660f-d473-400a-88ad-52d55f3fc9f5",
      "label": [
        {
          "isoCode": "fra",
          "value": "Observatoire d’Hyrule",
        },
      ],
      "parcels": [
        "",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2020-07-03T00:00:00.000Z",
    },
    "17ec2d21-8c66-4a43-921e-817fb489f899": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.06278,
          48.759994,
        ],
        "type": "Point",
      },
      "id": "17ec2d21-8c66-4a43-921e-817fb489f899",
      "label": [
        {
          "isoCode": "fra",
          "value": "Les Quatre Maisons",
        },
      ],
      "parcels": [
        "",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2022-08-23T00:00:00.000Z",
    },
    "26d79f86-7051-4c51-a1b1-966fd7f1e345": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.060606,
          48.768683,
        ],
        "type": "Point",
      },
      "id": "26d79f86-7051-4c51-a1b1-966fd7f1e345",
      "label": [
        {
          "isoCode": "fra",
          "value": "Armurerie de l’Ancien Royaume",
        },
      ],
      "parcels": [
        "",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2022-06-13T00:00:00.000Z",
    },
    "3c091079-e2c0-4151-8d1f-3a000bc29894": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.061001,
          48.761039,
        ],
        "type": "Point",
      },
      "id": "3c091079-e2c0-4151-8d1f-3a000bc29894",
      "label": [
        {
          "isoCode": "fra",
          "value": "Empreinte du Fleau",
        },
      ],
      "parcels": [
        "",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2020-08-31T00:00:00.000Z",
    },
    "413eb0cb-561a-435a-af8f-d244023f9b40": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.081033,
          48.772203,
        ],
        "type": "Point",
      },
      "id": "413eb0cb-561a-435a-af8f-d244023f9b40",
      "label": [
        {
          "isoCode": "fra",
          "value": "La voie celeste",
        },
      ],
      "parcels": [
        "",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2020-07-03T00:00:00.000Z",
    },
    "468d0b9b-ec0c-448d-8188-4e28762251e2": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.082305,
          48.762015,
        ],
        "type": "Point",
      },
      "id": "468d0b9b-ec0c-448d-8188-4e28762251e2",
      "label": [
        {
          "isoCode": "fra",
          "value": "Route de la Baleine",
        },
      ],
      "parcels": [
        "212860000D0053,212860000D0054",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "787ca7cf-8072-47ae-a8c6-98a62a8dd90c": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.081444,
          48.769615,
        ],
        "type": "Point",
      },
      "id": "787ca7cf-8072-47ae-a8c6-98a62a8dd90c",
      "label": [
        {
          "isoCode": "fra",
          "value": "Rue Rhoam Bosphoramus",
        },
      ],
      "parcels": [
        "212860000C0237,212860000C0107",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "7ce61747-d840-4019-97be-82dc0568619f": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.085574,
          48.774035,
        ],
        "type": "Point",
      },
      "id": "7ce61747-d840-4019-97be-82dc0568619f",
      "label": [
        {
          "isoCode": "fra",
          "value": "Allée des Prodiges",
        },
      ],
      "parcels": [
        "",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "8b3ae0a2-81ad-45b6-b58a-925868e6a3bc": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.060016,
          48.785677,
        ],
        "type": "Point",
      },
      "id": "8b3ae0a2-81ad-45b6-b58a-925868e6a3bc",
      "label": [
        {
          "isoCode": "fra",
          "value": "Passage de Daruk",
        },
      ],
      "parcels": [
        "",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2020-07-03T00:00:00.000Z",
    },
    "c965715f-3874-4cba-ae4e-c9afa585f5eb": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.079699,
          48.773594,
        ],
        "type": "Point",
      },
      "id": "c965715f-3874-4cba-ae4e-c9afa585f5eb",
      "label": [
        {
          "isoCode": "fra",
          "value": "Impasse des lynels",
        },
      ],
      "parcels": [
        "",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "c9b6df77-638b-4b30-991e-71486a91ea95": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.087958,
          48.760488,
        ],
        "type": "Point",
      },
      "id": "c9b6df77-638b-4b30-991e-71486a91ea95",
      "label": [
        {
          "isoCode": "fra",
          "value": "Chemin de la legende",
        },
      ],
      "parcels": [
        "212860000D0143",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2021-09-06T00:00:00.000Z",
    },
    "cfdb5a41-75b4-4964-ad95-f14f4ab6b23f": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.088235,
          48.758813,
        ],
        "type": "Point",
      },
      "id": "cfdb5a41-75b4-4964-ad95-f14f4ab6b23f",
      "label": [
        {
          "isoCode": "fra",
          "value": "Etang du sorcier",
        },
      ],
      "parcels": [
        "",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2020-07-03T00:00:00.000Z",
    },
    "d447b2d5-ff6d-42bc-8cb2-7efbfa2030e7": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.071619,
          48.751484,
        ],
        "type": "Point",
      },
      "id": "d447b2d5-ff6d-42bc-8cb2-7efbfa2030e7",
      "label": [
        {
          "isoCode": "fra",
          "value": "Arbre Mojo",
        },
      ],
      "parcels": [
        "",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2020-07-03T00:00:00.000Z",
    },
    "e89808c4-7f7e-46f3-b3a8-d40e4b783d8d": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.077283,
          48.766076,
        ],
        "type": "Point",
      },
      "id": "e89808c4-7f7e-46f3-b3a8-d40e4b783d8d",
      "label": [
        {
          "isoCode": "fra",
          "value": "Chemin du Moulin",
        },
      ],
      "parcels": [
        "",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2023-01-13T00:00:00.000Z",
    },
    "f52cf4c8-df1d-4619-9a64-0b0b3ee01c4e": {
      "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
      "geometry": {
        "coordinates": [
          1.071305,
          48.760311,
        ],
        "type": "Point",
      },
      "id": "f52cf4c8-df1d-4619-9a64-0b0b3ee01c4e",
      "label": [
        {
          "isoCode": "fra",
          "value": "Relai d’Hylia",
        },
      ],
      "parcels": [
        "",
      ],
      "type": {
        "value": "voie",
      },
      "updateDate": "2020-07-03T00:00:00.000Z",
    },
  },
  "districtID": "e2b5c142-3eb3-4d07-830a-3d1a59195dfd",
}
`;
